snap/snapcraft.yaml

```diff
@@ -234,6 +236,7 @@ parts:
       - python3-boto3
       - patroni=3.1.2-0ubuntu0.22.04.1~ppa2
       - golang-github-woblerr-pgbackrest-exporter=0.16.2+ds1-0ubuntu0.22.04.1~ppa1
+      - postgresql-ldap-sync=0.2.1-0ubuntu0.25.04.1~ppa3
```

Comment:
The deb package name is [python3-postgresql-ldap-sync](https://launchpad.net/~data-platform/+archive/ubuntu/postgresql-ldap-sync/+files/python3-postgresql-ldap-sync_0.2.1-0ubuntu0.25.04.1~ppa3_all.deb)

Reply:
Switched to the correct package.